### PR TITLE
Add DBus structures for comps data

### PR DESCRIPTION
**Pull request by poncovka**
Use the `CompsGroupData` and `CompsEnvironmentData` data structures to get
information about groups and environments defined by `comps.xml`.

Use the DNF manager to get information about available groups and environments.

### CODE REVIEW

File: tests/nosetests/pyanaconda_tests/modules/payloads/payload/module_payload_dnf_manager_test.py [hunk 1]
```diff
@@ -553,3 +539,207 @@ def download_location_test(self):
 
         self.dnf_manager.reset_base()
         self.assertEqual(self.dnf_manager.download_location, None)
+
+
+class DNFMangerCompsTestCase(unittest.TestCase):
```
Comment (rvykydal) [inline tests/nosetests/pyanaconda_tests/modules/payloads/payload/module_payload_dnf_manager_test.py:544]: a typo DNFManger

**Review by rvykydal — APPROVED** [review]
Looks good to me.

File: tests/nosetests/pyanaconda_tests/modules/payloads/payload/module_payload_dnf_manager_test.py (hunk 1 above)
Comment (poncovka) [reply tests/nosetests/pyanaconda_tests/modules/payloads/payload/module_payload_dnf_manager_test.py]: Fixed.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by poncovka** [comment]
Rebased.

**Pull request merged**